### DISCUSSION TITLE
Who is using Inspektor Gadget? And how?

### DISCUSSION THREAD

**Discussion by mayasingh17** [inspektor-gadget/inspektor-gadget#3779] [General]
We'd love to hear who is using Inspektor Gadget and how you're using it.
What gadgets are you using it?
How are you using it? Kubectl gadget, Kubectl debug, ig, API?

Please specify if you're using Inspektor Gadget in Production. :)
If you are using it in Prod please update our adopters file: https://github.com/inspektor-gadget/inspektor-gadget/blob/main/ADOPTERS.md

We're looking for partnerships to work closely with to ensure Inspektor Gadget solves production use cases. If interested, please email us at [hello@inspektor-gadget.io](mailto:hello@inspektor-gadget.io)

**Comment by PixelRobots** [comment]
I used Inspektor Gadget in production via the VS Code AKS extension to troubleshoot a DNS issue. We were experiencing a large number of failed DNS entries, and I needed to pinpoint the source. Using Inspektor Gadget, I identified a specific pod responsible for the problem. This allowed me to resolve the issue quickly without extensive trial-and-error debugging.

The integration with the [AKS VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) made it easy to access and use, streamlining the troubleshooting process significantly.
